pkg/omf/cli/omf.repo.pull.fish

```diff
@@ -1,42 +1,48 @@
 function omf.repo.pull
-  if test (command git config --get remote.upstream.url)
+  if test (count $argv) -eq 0
```

Comment:
As this is a function used only by `omf` I wouldn't bother adding a check for arguments. What do you think about this?